dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -2584,27 +2580,6 @@ def test_min_max_id_properly_update(self, *mocks):
         self.assertEqual(0, profile._min_id)
         self.assertEqual(6, profile._max_id)
 
-    @mock.patch(
```

Comment:
This tests was partially moved. It's functionality is tested elsewhere. (lines 3817-3823, 4076-4152)

Reply:
It doesn't make sense for this functionality to be in `TestStructuredColProfilerClass`

Reply:
Yup, weird it was in the `TestStructuredColProfilerClass` class to begin with...